tests/dialects/test_stencil.py

```diff
@@ -513,42 +486,7 @@ def test_stencil_temptype_constructor(attr: IntegerType, dims: tuple[int]):
 def test_stencil_temptype_constructor_empty_list(attr: IntegerType, dims: list[int]):
     with pytest.raises(VerifyException) as exc_info:
         TempType(dims, attr)
-    assert (
-        exc_info.value.args[0]
-        == "Number of field dimensions must be greater than zero, got 0."
-    )
-
-
-@pytest.mark.parametrize(
-    "attr, dims",
-    (
-        (i32, [1, 2]),
-        (i32, [1, 1, 3]),
-        (i64, [1, 1, 3]),
-    ),
-)
-def test_stencil_temptype_printing(attr: IntegerType, dims: list[int]):
```

Comment:
Do we want to remove tests for printing?

Reply:
Hey, this is removing the test (and only use) of `__repr__`, not the actual printing of the types as in textual representation.
This was probably a workaround to improve error reporting?

Reply:
And I removed the method itself, not just the test, that's why! Do you know of any specific use it had maybe?

Reply:
> Do you know of any specific use it had maybe?

I do not have any specific use case for it in my mind atm. 

Do we want to add test(s) in this PR to hit [this printing function](https://github.com/xdslproject/xdsl/blob/7776476bfad969ec7c3601a1aad26b135cc596d1/xdsl/dialects/experimental/stencil.py#L136) for improving coverage?

Reply:
Good then, thanks!